modules/rasBidAdapter.js

```diff
@@ -0,0 +1,136 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { isEmpty, getAdUnitSizes, parseSizesInput, deepAccess } from '../src/utils.js';
+
+const BIDDER_CODE = 'ras';
+const GET_ENDPOINT = (network) => `https://csr.onet.pl/${encodeURIComponent(network)}/csr-006/csr.json?`;
+const VERSION = '1.0';
+
+function parseParams(params) {
+  const newParams = {};
+  const pageContext = params.pageContext;
+  if (!pageContext) {
+    return {};
+  }
+  if (pageContext.dr) {
+    newParams.dr = pageContext.dr
```

Comment:
`bidderRequest.refererInfo.referer` must be checked in addition to a bidder-specific parameter looking for the referrer URL.

Reply:
it is .page now

Reply:
@lksharma: pull request is updated. Now we obtain `bidderRequest.refererInfo.referer` value if `pageContext.du` custom param is not provided.

@patmmccann: I don't see `bidderRequest.refererInfo.page` value in pbjs v6.29.0-pre. Has it been introduced in a different release?

Reply:
@patmmccann are you suggesting using `ortb2.site.page` or `ortb2.site.ref` over `bidderRequest.refererInfo.referer`?

Reply:
@skoklowski As of prebid 7.0.0 `bidderRequest.refererInfo` has the following interfaces:

1. page - the best candidate for the current page URL
2. ref - the referrer (document.referrer) to the current page, or null if not available (due to cross-origin restrictions)

Please update the current implementation the use the one most appropriate for your case. 

FYI - https://docs.prebid.org/dev-docs/bidder-adaptor.html#building-the-request:~:text=AdUnit.mediaType.video-,Location%20and%20referrers,-Referrer%20information%20should

Reply:
@lksharma I have updated the implementation to use the new interface